.github/workflows/chrome_v112.yml

```
@@ -0,0 +1,20 @@
+name: chrome_v112
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          cache: "yarn"
+      - uses: browser-actions/setup-chromium@v1
+        with:
+          chrome-version: "1109224"
```

Uses closest for v112, as 1109220 artifact is missing
From https://chromiumdash.appspot.com/releases?platform=Linux

```suggestion
          # Closest available commit position to official v112 from From https://chromiumdash.appspot.com/releases?platform=Linux
          chrome-version: "1109220"
```